includes/class-wp-job-manager-install.php

```diff
@@ -76,6 +76,8 @@ public static function install() {
 			update_option( 'job_manager_permalinks', wp_json_encode( $permalink_options ) );
 		}
 
+		\WP_Job_Manager\Stats::instance()->migrate();
```

Comment:
Does it make sense to move this call to the activate() method of Stats instead? Note that this will mean that migrate will be called earlier, but I think it is ok to do so.

Reply:
I don't think that makes sense, @pgk. Because `activate()` AFAIK will not run when the plugin is updated (e.g. from an old WPJM installation), while this method will be run both on new plugin installations as well as plugin upgrades.

Does that make sense?